### ISSUE TITLE
[IMPROVEMENT] Please ADD "Version" & "New Version" Columns to the "Managed Ignored Updates" LIST.

### ISSUE THREAD

**Issue by leeuniverse** [marticliment/UniGetUI#2957]
### Please confirm these before moving forward.

- [X] I have searched for my feature proposal and have not found a work-in-progress/duplicate/resolved/discarded issue.
- [X] This improvement refers to an existing feature. If you want to suggest a new feature, please use [this template](https://github.com/marticliment/WingetUI/issues/new?assignees=marticliment&labels=new-feature&projects=&template=feature-request.yml&title=%5BFEATURE+REQUEST%5D+%28Enter+your+description+here%29).
- [X] This improvement is not a bug. If you want to report a bug, please use [this template](https://github.com/marticliment/WingetUI/issues/new?assignees=marticliment&labels=bug&projects=&template=bug-issue.yml&title=%5BBUG%5D+%28Enter+your+description+here%29).

### Describe the improvement

The Ignored Updates List needs to show the Current Version # and the Latest Version # also.

### Describe how this improvement could help users

How this helps is I have software in which I need to Update "Manually" due to special configurations.
So, I don't include these in the Update List, I add them to the Ignore Update List.

However, it would be VERY useful if we could open the Ignore List and SEE what of those Apps ALSO need an Update.
This would be very helpful in further tracking our Updates thus Updating our Apps....

Thank you...

**Comment by mrixner** [comment]
Is there any way to get the new version of an update-ignored package? I tried using `PEInterface.UpgradablePackagesLoader`, but `UpgradablePackagesLoader` filters out ignored packages before they ever reach the internal list (in `UpgradeablePackagesLoader.IsPackageValid`). I also tried `PEInterface.InstalledPackagesLoader`, but `InstalledPackagesLoader` doesn't seem to actually determine if there are updates to a package.

The only solution I could think of is having a separate list in `UpgradeablePackagesLoader` for ignored-update packages, which could then be queried in the dialog. To do this, either after `IsValid` is called and returns false I would check if it's an ignored update package and add it to the other list, but bypassing `IsValid` doesn't seem like a great strategy. The other thing I thought of was to change the `AbstractPackageLoader.IsValid` template to optionally (based on a parameter passed to the function) mark ignored-update packages as valid and then manually check in `AbstractPackageLoader.ReloadPackages` if it's an update-ignored package and sort it accordingly, but this would require changing the `IsValid` base function for something that doesn't affect all of the package loaders.
